lib/stat_tracker.rb

```diff
@@ -51,4 +51,42 @@ def best_season(id)
         "20172018"
       end
     end
+
+    def worst_season(id)
```

Comment:
This method is way too long, we'll have to cut it down somehow. I'll need some help/ideas from you fellas